assets/js/modules/reader-revenue-manager/components/dashboard/RRMSetupSuccessSubtleNotification.js

```diff
@@ -200,7 +203,13 @@ export default function RRMSetupSuccessSubtleNotification( {
 	] );
 
 	const gaTrackingProps = {
-		gaTrackingEventArgs: { label: publicationOnboardingState },
+		gaTrackingEventArgs: {
+			label: publicationOnboardingState,
+			category: trackEventCategory,
+			value: `${ publicationOnboardingState }:${ paymentOption }:${
+				productID ? 'yes' : 'no'
+			}`,
```

Comment:
Similar to other GA tracking instances on our codebase, we do not want to use the `value` property. We just want to update the `label` property.

```suggestion
```